### DISCUSSION TITLE
Inferencer Code Editing

### DISCUSSION THREAD

**Discussion by PriyanshuBBS** [refinedev/refine#5052] [Q&A] [answered]
How to edit the code that is generated by it. As i am beginner, couldn't understand the documentation which i found quite complex. So, Kindly explain me how to edit the code or to disable inferencer. Thanks a lot

**Comment by BatuhanW** [comment, marked as answer]
Hello @PriyanshuBBS, inferencer makes a `guess` and generates some code for you. Then you should copy the generated code and paste it into respective component. For example, if you are in `posts` route, which is `PostsList` component, you can paste inferencer code into this component and modify as you need.

**Reply by PriyanshuBBS** [reply]
Thanks a lot for help, @BatuhanW
